pom.xml

```diff
@@ -125,11 +125,6 @@
                 <artifactId>jetty-jaas</artifactId>
                 <version>${jetty.version}</version>
             </dependency>
-            <dependency>
```

Comment:
presumably we still want the dependency, just not override the version?

Reply:
Actually we do want to remove the entire dependency here in the dependencymanagement section of the repo pom. Otherwise the dependency specified in the common pom gets overridden. The core submodule's pom still references the jackson-databind dependency, and inherits correctly from the common pom.

Reply:
ah, i missed that this was in dependencyManagement, I thought it was in dependencies.